root-pages/principles.md

```diff
@@ -0,0 +1,84 @@
+---
+displayed_sidebar: null
+sidebar_class_name: hidden
+pagination_prev: null
+pagination_next: null
+---
+
+# Principles
```

Comment:
What is that?

Reply:
Did you mean to copy https://vac.dev/principles ?

Reply:
Yes content is from https://raw.githubusercontent.com/vacp2p/vac.dev/refs/heads/develop/docs/principles.md

Reply:
it's not though...